libs/ui/icon/helm/src/lib/hlm-icon.component.ts

```diff
@@ -36,7 +36,8 @@ export const iconVariants = cva('inline-flex', {
 	},
 });
 
-export type IconSize = DefinedSizes | string;
+// eslint-disable-next-line @typescript-eslint/ban-types
+export type IconSize = DefinedSizes | ({} & string);
```

Comment:
I suggest being slightly more explicit with:
```suggestion
export type IconSize = DefinedSizes | (Record<never, never> & string);
```
Or better yet, we could install a type utility library like [type-fest](https://github.com/sindresorhus/type-fest) and use [LiteralUnion](https://github.com/sindresorhus/type-fest/blob/main/source/literal-union.d.ts) (which I referred to confirm your change).

```suggestion
export type IconSize = LiteralUnion<DefinedSizes>;
```
But I'm not sure if @goetzrobin would want to add the library for just one type helper.

Reply:
I changed it to `Record<never, never>`

If @goetzrobin says I can introduce a new dependency like the one you mentioned, but in general I am fine with this.